packages/react/src/context/slash-id-context.tsx

```diff
@@ -52,13 +54,14 @@ export interface SlashIDProviderProps {
    */
   sdkUrl?: string;
   analyticsEnabled?: boolean;
+  anonymousUsersEnabled?: boolean;
```

Comment:
It would be useful to annotate this property with a TSDoc style comment, this will pop up in the editor - could save someone a visit to the docs.

Reply:
done